Extensions/ParticleSystem/particleemitterobject.ts

```diff
@@ -88,10 +84,10 @@ namespace gdjs {
     _colorDirty: boolean = true;
     _sizeDirty: boolean = true;
     _alphaDirty: boolean = true;
-    _textureDirty: boolean;
-
-    // Don't mark texture as dirty if not using one.
```

Comment:
Nice, these comments got their location messed up a while ago when codemoding from JS+JSdoc to full TypeScript.